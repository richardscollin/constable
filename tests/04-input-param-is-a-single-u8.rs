use constable::lookup;

#[lookup]
const fn foo(_x: u16) -> bool {
    false
}

fn main() {}
