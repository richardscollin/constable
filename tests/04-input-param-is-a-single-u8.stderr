error: custom attribute panicked
 --> tests/04-input-param-is-a-single-u8.rs:3:1
  |
3 | #[lookup]
  | ^^^^^^^^^
  |
  = help: message: input param isn't a single u8
